tools/testdata/protoxform/envoy/v2/BUILD

```diff
@@ -15,3 +15,18 @@ proto_library(
         "@envoy_api//envoy/api/v2:pkg",
     ],
 )
+
+proto_library(
+    name = "freeze_protos",
+    srcs = [
+        "active_non_terminal.proto",
+        "active_terminal.proto",
+        "frozen.proto",
```

Comment:
None of the previous 3 proto source files exist.  This breaks compilation of //tools/...